ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -159,49 +134,18 @@ const styles = reactStyles({
     marginLeft: '-1rem',
     paddingTop: '0.3rem',
     paddingBottom: '0.3rem'
+  },
+  spinner: {
+    position: 'fixed',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    backgroundColor: 'rgba(0, 0, 0, 0.2)',
+    height: '100%',
+    width: '100%',
   }
 });
 
-export const WorkspaceEditSection = (props) => {
```

Comment:
Moved WorkspaceEditSection along with WorkspaceResearchSummary object to another file since there is already a lot going on in workspace-edit file.